docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx

```diff
@@ -156,31 +164,52 @@ refiner = DiffusionPipeline.from_pretrained(
 refiner.to("cuda")
 ```
 
-Now we define the number of inference steps and the fraction at which the model shall be run through the 
+Now we define the number of inference steps and the point at which the model shall be run through the 
 high-noise denoising stage (*i.e.* the base model).
 
 ```py
 n_steps = 40
-high_noise_frac = 0.7
+high_noise_transition = 200  # or high_noise_frac = 0.8
 ```
 
-A fraction of 0.7 means that 70% of the 40 inference steps (28 steps) are run through the base model
-and the remaining 12 steps are run through the refiner. Let's run the two pipelines now.
-Make sure to set `denoising_end` and `denoising_start` to the same values and keep `num_inference_steps`
-constant. Also remember that the output of the base model should be in latent space:
+Stable Diffusion XL base is trained on timesteps 0-999 and Stable Diffusion refiner is finetuned
+from the base on timesteps 0-200, so we use the base model for the first 800 timesteps (high noise) 
+and the refiner for the last 200 timesteps (low noise). Hence, `high_noise_transition`
+is set to 200, so that all steps 200-999 are performed by the base model and steps 0-199 are
+performed by the refiner model.
+
+Alternatively, a fraction of 0.8 (`high_noise_transition`) means that 80% of the 40 inference steps (32 steps) are
+run through the base model and the remaining 8 steps are run through the refiner when
+`use_karras_sigmas=False`, with the base model handling timesteps 201-999 while the refinement
+model handles timesteps 0-200. **Please note this timestep proportional relationship does not hold true when
+using the `use_karras_sigmas` option and you should instead use an integer value as described above.**
```

Comment:
Hmm I see that there's a difference of 1 in the timesteps. In the "int" setting, the brackets are (0-199) and (200 - 999) but here it's (0, 200) and (201-999).

Reply:
Correct, see: https://github.com/huggingface/diffusers/pull/4115#issuecomment-1638433779

IMO: the float proportion is buggy, but since you already merged it I didn't want to make backwards incompatible changes.